data/src/main/java/camp/nextstep/edu/github/data/DefaultGitHubRepository.kt

```diff
@@ -0,0 +1,12 @@
+package camp.nextstep.edu.github.data
+
+import camp.nextstep.edu.github.domain.GithubRepository
+import camp.nextstep.edu.github.domain.Repositories
+
+internal class DefaultGitHubRepository(
+    private val gitHubService: GitHubService
+) : GithubRepository {
+    override suspend fun loadRepositories(): Repositories = gitHubService.listRepos()
+        .map { it.toDomain() }
```

Comment:
```suggestion
        .map(RepoModel::toDomain)
```
전달하는 파라미터 타입의 개수와 숫자가 일치할 경우에는, 메서드 참조를 활용할 수 있어요.